pkg/kn/commands/revision/describe.go

```diff
@@ -274,3 +282,28 @@ func stringifyEnv(revision *v1alpha1.Revision) []string {
 	}
 	return envVars
 }
+
+func stringifyEnvFrom(revision *v1alpha1.Revision) []string {
+	container, err := clientserving.ContainerOfRevisionSpec(&revision.Spec)
+	if err != nil {
+		return nil
+	}
+	var configMap []string
+	var secret []string
+	var result []string
+	for _, envFromSource := range container.EnvFrom {
+		if envFromSource.ConfigMapRef != nil {
+			configMap = append(configMap, envFromSource.ConfigMapRef.Name)
+		}
+		if envFromSource.SecretRef != nil {
+			secret = append(secret, envFromSource.SecretRef.Name)
+		}
+	}
+	if len(configMap) > 0 {
+		result = append(result, fmt.Sprintf("ConfigMap:(%s)", strings.Join(configMap, ",")))
```

Comment:
Can we please use the same format as it can be specified on the command line ? Ie. "cm:mymap" an "secret:mysecret" ? This helps to keep our wording consistent. (and you can better reuse it with copy&paste actions).

Reply:
OK.